### PR TITLE
Added support for LocalStorage configuration, initially for JsonSerializerOptions

### PR DESCRIPTION
**Pull request by jan-johansson-mr**
Hi,

I've added support for tweaking json serializer options for LocalStorage. I've updated the tests for the change of the code. Please review and you're welcome with any feedback.

Thanks,
//Jan

### CODE REVIEW

**Comment by jan-johansson-mr** [comment]
Hi,

I've let the type of the object to be exposed to the serializer (from the former behavior when 'object' was passed to the serializer). The reason for this is that I have a converter that actually is triggered by the type in the graph to be serialized, the passing of 'object' doesn't trigger my converter. The change will not break existing code, so the change is seamless.  Please check it out

Thanks,
//Jan

File: src/Blazored.LocalStorage/Blazored.LocalStorage.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <RootNamespace>Blazored.LocalStorage</RootNamespace>
 
     <PackageId>Blazored.LocalStorage</PackageId>
-    <Version>2.1.5</Version>
+    <Version>2.1.6-preview-0.0.2</Version>
```
Comment (chrissainty) [inline src/Blazored.LocalStorage/Blazored.LocalStorage.csproj:9]: Could you undo this one? We set the build number when this goes to release.

Comment (jan-johansson-mr) [reply src/Blazored.LocalStorage/Blazored.LocalStorage.csproj]: No problems :-)

**Comment by jan-johansson-mr** [comment]
Added documentation and reverted version tag in project file

**Comment by jan-johansson-mr** [comment]
Hey,

I've ripped off your amazing LocalStorage and created [SessionStorage](https://github.com/jan-johansson-mr/SessionStorage). Please take a look and I'd be happy if you included this derivative work alongside with LocalStorage.

Thanks,
//Jan

**Comment by chrissainty** [comment]
Hi Jan you there is already [Blazored Session Storage](https://github.com/Blazored/SessionStorage)

**Comment by jan-johansson-mr** [comment]
Oh, I missed that one, since it was not pinned :-) Thanks!

**Pull request merged**